src/revoked/verify.test.js

```diff
@@ -0,0 +1,227 @@
+const { isRevoked } = require("./contractInterface");
+const {
+  isAnyHashRevokedOnStore,
+  revokedStatusOnContracts,
+  isRevokedOnAny,
+  getIntermediateHashes,
+  verifyRevoked
+} = require("./verify");
+
+jest.mock("./contractInterface");
+
+beforeEach(() => {
+  isRevoked.mockClear();
+});
+
+const TOKEN_REGISTRY_CONTRACT = {
+  address: "0x0A",
+  instance: "CONTRACT_INSTANCE",
+  type: "TOKEN_REGISTRY"
+};
+
+const DOCUMENT_STORE_CONTRACT = {
+  address: "0x0B",
+  instance: "CONTRACT_INSTANCE",
+  type: "DOCUMENT_STORE"
+};
+
+const INTERMEDIATE_HASHES = ["0x0a", "0x0b", "0x0c"];
+
+describe("isAnyHashRevokedOnStore", () => {
+  it("returns false if none of the hash are revoked", async () => {
+    isRevoked.mockResolvedValue(false);
+    const revoked = await isAnyHashRevokedOnStore(
+      TOKEN_REGISTRY_CONTRACT,
+      INTERMEDIATE_HASHES
+    );
+    expect(revoked).toBe(false);
+  });
+
+  it("returns true if any of the hashes is revoked", async () => {
+    isRevoked.mockResolvedValueOnce(false);
+    isRevoked.mockResolvedValueOnce(true);
+    isRevoked.mockResolvedValueOnce(false);
+    const revoked = await isAnyHashRevokedOnStore(
+      TOKEN_REGISTRY_CONTRACT,
+      INTERMEDIATE_HASHES
+    );
+    expect(revoked).toBe(true);
+  });
+});
+
+describe("revokedStatusOnContracts", () => {
+  it("returns a mapping of smart contract to revoke status", async () => {
+    const smartContracts = [TOKEN_REGISTRY_CONTRACT, DOCUMENT_STORE_CONTRACT];
```

Comment:
do you need to set isRevoked.mockResolvedValue for this test?

Reply:
Good catch, interesting that `mockClear` clears the status but not the return values, so it was still resolving from two test above.